packages/design-docs/src/content/Conventions.mdx

```diff
@@ -63,7 +63,7 @@ Check this NN/G group article about the impact of tone of voice on the brand's p
 
 ## Spelling: American vs. British
 
-Follow spelling rules for American English. Check this site to get some help about your word choice: https://www.oxfordinternationalenglish.com/differences-in-british-and-american-spelling/
+Follow spelling rules for American English. Check this site to get some help about your word choice: [https://www.oxfordinternationalenglish.com/differences-in-british-and-american-spelling/](https://www.nngroup.com/articles/tone-voice-users/)
```

Comment:
this looks like hacker to display a link but pointing outside :thinking:

Reply:
oupsi